apps/web-portal-client/lib/components/nav-bar/LoginButton.jsx

```diff
@@ -1,28 +1,29 @@
 import React from 'react'
-import PropTypes from 'prop-types'
+import {useDispatch} from 'react-redux'
+import {userLogout} from '@web-portal-client/features/auth/effects'
```

Comment:
just a reminder, let's rid-off thous aliases, yet one issue is an IDE code highlighting

Reply:
done